RELEASE.md

```diff
@@ -104,12 +118,18 @@ Entries in the `CHANGELOG.md` are meant to be in this order:
 
 Create a PR for the changes to be reviewed.
 
+You can use github UI to get differences between commits in latest release branch and previous released version.
+
+eg: https://github.com/prometheus-operator/prometheus-operator/compare/v0.72.0...release-0.73
+
 ## Publish the new release
 
 For new minor and major releases, create the `release-<major>.<minor>` branch starting at the PR merge commit.
 Push the branch to the remote repository with
 
-```
+**Note:** The remote name `origin` is assumed to be pointed to `prometheus-operator/prometheus-operator`. If you have a different remote name, use that instead of `origin`. Verify this using `git remote -v`
```

Comment:
```suggestion
**Note:** The remote name `origin` is assumed to be pointed to `github.com/prometheus-operator/prometheus-operator`. If you have a different remote name, use that instead of `origin`. Verify this using `git remote -v`
```